R/tt_compatibility.R

```diff
@@ -374,6 +391,19 @@ rbindl_rtables <- function(x, gap = 0, check_headers = TRUE) {
 
     lapply(x, function(xi) chk_compat_cinfos(x[[1]], xi)) ##col_info(xi)))
 
+    rbind_annot <- list(title = "", subtitles = character(), footer = character(), pf = character())
+    # Titles/footer info are (independently) retained from first object if identical or missing in all other objects
+    all_titles <- sapply(x, main_title)
+    if (all_annots_identical(all_titles) || only_first_annot(all_titles)) rbind_annot[["title"]] <- all_titles[[1]]
```

Comment:
This if statement (and those below it) aren't short enough to justify the single line form, please move the consequent (the thing that happens if the condition is true) to the next line here and below.